src/Tools/EnvironmentUtility.php

```diff
@@ -100,6 +101,9 @@ public function getInstalledPackageVersion(string $packageName): ?string
     {
         $contents = file_get_contents($this->spotVendorPath() . '/composer/installed.json') ?: '';
         $activeInstallations = json_decode($contents, false);
+        if (!($activeInstallations instanceof stdClass)) {
```

Comment:
under what circumstance will this happen?

Reply:
phpstan